### PR TITLE
add support for `polkadot-sdk`

### PR DESCRIPTION
**Pull request by muraca**
This PR adds support for the Polkadot-SDK monorepo by introducing the flag `--sdk` in both patch and update commands.

### CODE REVIEW

**Comment by muraca** [comment]
up @bkchr 🙂

**Comment by lemunozm** [comment]
Hi, is this change expected to be merged soon?

**Comment by muraca** [comment]
@lemunozm I don't think so. 
You can just checkout my fork and launch `cargo build --release`, then replace your local binary

File: src/patch.rs [hunk 1]
```diff
@@ -106,46 +106,53 @@ pub struct Patch {
...
+        conflicts_with_all = &[ "crates", "substrate", "cumulus", "polkadot", "beefy", "sdk" ]
     )]
     target: Option<String>,
 
+    /// Use the official Polkadot-SDK monorepo as patch target.
+    #[structopt(
+    long,
+    conflicts_with_all = &[ "target", "substrate", "polkadot", "cumulus", "beefy", "crates" ]
+)]
+    sdk: bool,
```
Comment (bkchr) [inline src/patch.rs:118]: We don't need `substrate`, `polkadot` etc anymore as everything is now in the mono repo. Can you remove them?

**Review by bkchr — COMMENTED** [review]
Sorry for having overseen this!

File: src/patch.rs (hunk 1 above)
Comment (muraca) [reply src/patch.rs:118]: sure, will do asap

Comment (muraca) [reply src/patch.rs:118]: well, ditching support for all old repositories would only leave one possible workflow, therefore I will remove the need for any option and just support `polkadot-sdk`

Comment (bkchr) [reply src/patch.rs:118]: There is still the `crates` option?

Comment (muraca) [reply src/patch.rs]: yes, I just pushed the changes

**Review by bkchr — APPROVED** [review]

**Pull request merged**